packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -8,7 +19,8 @@ const typesToEmbed: Record<EmbedType, string> = {
   'side-tab': 'popup-side-panel',
 }
 
-const mapOptionsToQueryParams = (type: EmbedType, options: UrlOptions): Record<string, any> => ({
+const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlOptions): Record<string, any> => ({
```

Comment:
Lets keep all the logic of mapping options (or other values) to query params in one method.